src/Core/src/Handlers/View/ViewHandler.Windows.cs

```diff
@@ -0,0 +1,48 @@
+﻿#nullable enable
+using NativeView = Microsoft.UI.Xaml.FrameworkElement;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ViewHandler
+	{
+		public static void MapTranslationX(IViewHandler handler, IView view) 
+		{ 
+			((NativeView?)handler.NativeView)?.UpdateRotation(view);
```

Comment:
Thi says rotation here, but maybe it needs to be like iOS and just have one big map? Seems everything eventually calls this - even UpdateScaleAndRotation.